diff -up prosody-0.8.2/Makefile.patch prosody-0.8.2/Makefile
--- prosody-0.8.2/Makefile.patch	2013-04-27 23:21:21.703446153 +0200
+++ prosody-0.8.2/Makefile	2013-04-27 23:22:19.525442335 +0200
@@ -20,7 +20,6 @@ install: prosody.install prosodyctl.inst
 	install -d $(BIN) $(CONFIG) $(MODULES) $(SOURCE)
 	install -m750 -d $(DATA)
 	install -d $(MAN)/man1
-	install -d $(CONFIG)/certs
 	install -d $(SOURCE)/core $(SOURCE)/net $(SOURCE)/util
 	install -m755 ./prosody.install $(BIN)/prosody
 	install -m755 ./prosodyctl.install $(BIN)/prosodyctl
@@ -33,18 +32,16 @@ install: prosody.install prosodyctl.inst
 	install -m644 plugins/*.lua $(MODULES)
 	install -d $(MODULES)/muc
 	install -m644 plugins/muc/* $(MODULES)/muc
-	install -m644 certs/* $(CONFIG)/certs
 	install -d $(MODULES)/adhoc
 	install -m644 plugins/adhoc/*.lua $(MODULES)/adhoc
 	install -m644 man/prosodyctl.man $(MAN)/man1/prosodyctl.1
-	test -e $(CONFIG)/prosody.cfg.lua || install -m644 prosody.cfg.lua.install $(CONFIG)/prosody.cfg.lua
+	test -e $(CONFIG)/prosody.cfg.lua || install -m644 prosody.cfg.lua.dist $(CONFIG)/prosody.cfg.lua
 	test -e prosody.version && install prosody.version $(SOURCE)/prosody.version || true
 	$(MAKE) install -C util-src
 
 clean:
 	rm -f prosody.install
 	rm -f prosodyctl.install
-	rm -f prosody.cfg.lua.install
 	rm -f prosody.version
 	$(MAKE) clean -C util-src
 
