diff -up prosody-0.9.0beta1/Makefile.orig prosody-0.9.0beta1/Makefile
--- prosody-0.9.0beta1/Makefile.orig	2013-05-11 09:03:07.000000000 -0400
+++ prosody-0.9.0beta1/Makefile	2013-05-15 11:17:11.675317306 -0400
@@ -22,7 +22,6 @@ install: prosody.install prosodyctl.inst
 	install -d $(BIN) $(CONFIG) $(MODULES) $(SOURCE)
 	install -m750 -d $(DATA)
 	install -d $(MAN)/man1
-	install -d $(CONFIG)/certs
 	install -d $(SOURCE)/core $(SOURCE)/net $(SOURCE)/util
 	install -m755 ./prosody.install $(BIN)/prosody
 	install -m755 ./prosodyctl.install $(BIN)/prosodyctl
@@ -35,16 +34,14 @@ install: prosody.install prosodyctl.inst
 	install -d $(SOURCE)/util/sasl
 	install -m644 util/sasl/* $(SOURCE)/util/sasl
 	umask 0022 && cp -r plugins/* $(MODULES)
-	install -m644 certs/* $(CONFIG)/certs
 	install -m644 man/prosodyctl.man $(MAN)/man1/prosodyctl.1
-	test -e $(CONFIG)/prosody.cfg.lua || install -m644 prosody.cfg.lua.install $(CONFIG)/prosody.cfg.lua
+	test -e $(CONFIG)/prosody.cfg.lua || install -m644 prosody.cfg.lua.dist $(CONFIG)/prosody.cfg.lua
 	test -e prosody.version && install -m644 prosody.version $(SOURCE)/prosody.version || true
 	$(MAKE) install -C util-src
 
 clean:
 	rm -f prosody.install
 	rm -f prosodyctl.install
-	rm -f prosody.cfg.lua.install
 	rm -f prosody.version
 	$(MAKE) clean -C util-src
 
